docs/ember-intl/app/templates/docs/getting-started/quickstart-addons.md

```diff
@@ -1,6 +1,6 @@
 # Quickstart (Addons)
 
-Addons need to publish a folder called `translations`. Note, `translations` is the name that your apps expect, unless configured otherwise.
+Addons need to publish a folder called `translations`.
```

Comment:
"`translations` is the name that your apps expect, unless configured otherwise." is incorrect. For v1 addons, `translations` is the _only_ name that can be used.

I may restrict the possible name for Ember apps in `ember-intl@7.0.0`.